tests/core/math/test_vector3.h

```diff
@@ -35,7 +35,6 @@
 #include "tests/test_macros.h"
 
 #define Math_SQRT13 0.57735026918962576450914878050196
-#define Math_SQRT3 1.7320508075688772935274463415059
```

Comment:
Would it make sense to move Math_SQRT13 to [core/math/math_defs.h](https://github.com/godotengine/godot/pull/85890/files#diff-ba0b33ce06a1f988652d678b5b74ffc3e0833e56be56b5cb4b9838bc6237a0b4) as well?

~~*It must feel very sad and lonely for a constant to be only one in the file with no friends to talk to.*~~

Even if it's not used yet, it seems weird to have similar constants in two different places.